dist/localization.json

```diff
@@ -171,6 +173,7 @@
     "player_vehicle_spawned": "Spawnado {0}.",
     "player_give": "Dado {0} {2} {1}.",
     "discord": "Nosso Discord: {0}"
+    
```

Comment:
Please remove this newline if it doesn't have any use

Reply:
Done

Reply:
No?